adapters/handlers/rest/configure_api.go

```diff
@@ -697,6 +695,17 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 	return setupGlobalMiddleware(api.Serve(setupMiddlewares))
 }
 
+func StartBackupScheduler(appState *state.State) *backup.Scheduler {
```

Comment:
can we change this method to be a private method?

Reply:
It's exported for libweaviate